web/styles/app_components.css

```diff
@@ -626,7 +626,13 @@ input.settings_text_input {
 .unread_mention_info:not(:empty) {
     margin-right: 5px;
     margin-left: 2px;
-    opacity: 0.5;
+    opacity: 0.7;
+
+    .zulip-icon {
+        font-size: inherit;
+        position: relative;
```

Comment:
I'd really like us to get away from using positioning and offsets to position things. Are there better ways outside of `app_components.css` to work with or within grid or flexbox?

Reply:
You’re right, @karlstolley. However, my reasoning is that many similar icons are positioned using specific positioning and offsets. I felt this approach would keep things consistent with the existing code.
Also, this time, I didn’t make any changes to app_components.css. Instead, I wrote local CSS for the icons directly in their respective files.
Thanks!